packages/tailwindcss-react-aria-components/src/index.js

```diff
@@ -78,26 +78,35 @@ let mapSelector = (selector, fn) => {
   }
 };
 
-module.exports = plugin.withOptions((options) => (({addVariant}) => {
+let addVariants = (variantName, selectors, addVariant, matchVariant) => {
+  addVariant(variantName, selectors);
+  matchVariant(
+    'group',
+    (_, {modifier}) =>
+      modifier
+        ? mapSelector(selectors, selector => `:merge(.group\\/${modifier})${selector.slice(1)} &`)
+        : mapSelector(selectors, selector => `:merge(.group)${selector.slice(1)} &`),
+    {values: {[variantName]: variantName}}
+  );
+  addVariant(`peer-${variantName}`, mapSelector(selectors, selector => `:merge(.peer)${selector.slice(1)} ~ &`));
```

Comment:
Are there named peers too?

Reply:
Looks like it: https://tailwindcss.com/docs/hover-focus-and-other-states#differentiating-peers
I can update this to support those as well.